.travis.yml

```diff
@@ -1,14 +1,14 @@
+sudo: required
```

Comment:
Yeah, it's "discoverabilty" vs "predictability". Similar debate to using random vs fixed test data. No single or correct answer :)